0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,82 +1,91 @@
+The "Interledger protocol suite" can be used among any peered group of connectors, be it are public or private. There is no single network that all parties must connect to to use these protocols.
 
-"The Interledger" is a conceptual network made up of independent and diverse ledgers linked by connectors. Each account "on the Interledger" is part of a particular ledger, but they may transact with others by sending interledger payments through different ledgers and connectors. Like "the Internet", the Interledger is not a single network but is comprised of multiple interconnected networks.
+"The Interledger" is the name of a public instance of the Interledger protocol suite which seeks to provide a coherent global financial instructure. Ideally, anyone connected to the Interledger should be able to transact with anyone else, each using their currency and connector of choice.
 
 ## Interledger Security
 
-**Interledger uses *conditional transfers* to secure payments across multiple hops and even through untrusted connectors.** Senders are guaranteed cryptographic proof that the receiver got the payment or their money back, no matter how many ledgers and connectors are in between. Connectors take some risk, but this risk can be managed and is primarily based upon the connector's chosen ledgers and direct peers.
+**Interledger uses *conditional transfers* to secure payments across multiple hops and even through untrusted connectors.** Everyone only needs to trust their direct peers, no matter how many connectors are involved in forwarding a given packet. Connectors take some risk, but this risk can be managed and is primarily based upon the connector's chosen peers.
 
 <span class="show alert alert-info">**Hint:** Conditional transfers or *authorization holds* are the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
 
-### Conditional Transfers
+### Interledger Protocol Flow
 
-Each local transfer is first *prepared* and then either *executed* or *rejected*. When a transfer is prepared, the ledger puts the funds of the source account on hold with a *cryptographic condition* and *timeout*. If the condition is fulfilled before the timeout, the transfer is executed and the funds are transferred. If the timeout is reached, the transfer expires and the ledger returns the funds to the source account automatically.
+The sender first sends an Interledger `prepare` packet. Each connector may either forward the `prepare` packet, or, in the case of an error, return a `reject` packet. If the `prepare` packet receiver, they may respond with either a `fulfill` or `reject` packet, depending on whether they wish to accept the Interledger packet. In either case, the resulting packet is forwarded along the same path back to the sender.
```

Comment:
If the `prepare` packet [?] receiver -> missing "reaches the"?
accept the Interledger packet -> "accept the Interledger payment"?
forwarded along the same path back -> "passed along the same path back"?